.github/workflows/integrationTest.yml

```diff
@@ -499,4 +557,57 @@ jobs:
         run: terraform init
 
       - name: Terraform destroy
-        run: terraform destroy --auto-approve
\ No newline at end of file
+        run: terraform destroy --auto-approve
+
+  ECSFargateLinuxIntegrationTest:
+    name: 'StopLocalStack'
+    runs-on: ubuntu-latest
+    needs: [MakeDockerImage, GenerateTestMatrix]
+    strategy:
+      fail-fast: false
+      matrix:
+        arrays: ${{ fromJson(needs.GenerateTestMatrix.outputs.ecs_fargate_matrix) }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.TERRAFORM_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.TERRAFORM_AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+
+      - name: Cache if success
+        id: ecs-fargate-linux-integration-test
+        uses: actions/cache@v2
+        with:
+          path: RELEASE_NOTES
```

Comment:
Can we keep the path used here consistent? I see that above, we reference `go.mod` for the cache. I don't really care which one we use as long as we're consistent.

Reply:
I'm fine with both so yup.